app/tests/spec/views/settings/clients.js

```diff
@@ -503,6 +503,40 @@ define(function (require, exports, module) {
             assert.equal(formatted[0].name, 'client-1');
           });
       });
+
+      it('properly sets the title according to scope', function () {
+        return initView()
+          .then(() => {
+            $('#container').html(view.el);
+
+            const formatted = view._formatAccessTimeAndScope([
+              {
+                clientType: 'oAuthApp',
+                id: 'app-1',
+                lastAccessTime: Date.now(),
+                name: '123Done',
+                scope: ['profile']
+              },
+              {
+                clientType: 'oAuthApp',
+                id: 'app-2',
+                lastAccessTime: Date.now(),
+                name: 'Pocket',
+                scope: ['profile', 'profile:write']
+              },
+              {
+                clientType: 'oAuthApp',
+                id: 'app-3',
+                lastAccessTime: Date.now(),
+                name: 'Add-ons'
+              }
+            ]);
+
+            assert.equal(formatted[0].title, '123Done - profile');
```

Comment:
@divyabiyani this is great. we also need a quick assertion added to one of the View tests that the `title` attribute is there.